vessim/sil.py

```diff
@@ -126,7 +126,8 @@ def step(self, time: int, p_delta: float, actors: Dict) -> None:
         pipe.set("time", time)
         pipe.set("p_delta", p_delta)
         pipe.set("actors", json.dumps(actors))
-        pipe.set("microgrid", self.microgrid.pickle())
+        if self.microgrid is not None:
+            pipe.set("microgrid", self.microgrid.pickle())
```

Comment:
I changed it because originally in `controller.py`, microgrid is initalized as None:
```
class Controller(ABC):
  def __init__(self, step_size: int):
          self.microgrid = None
```
and then mypy throws an error for this line:  
````
pipe.set("microgrid", self.microgrid.pickle())
````
"vessim/sil.py:130: error: "None" has no attribute "pickle"  [attr-defined]"

Reply:
let's explicitly type it to `self. microgrid: Microgrid` because it's cannot be None at this point